cfme/tests/cloud/test_provisioning.py

```diff
@@ -47,14 +46,12 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
         'first_name': 'Image',
         'last_name': 'Provisioner',
         'notes': note,
-        'instance_type': provisioning['instance_type'] if provider.type != "azure" else
-        provisioning['vm_size'],
-        "availability_zone": provisioning["availability_zone"] if provider.type != "azure" else
-        None,
-        'security_groups': [provisioning['security_group']] if provider.type != "azure" else
-        provisioning["network_nsg"],
-        "guest_keypair": provisioning["guest_keypair"] if provider.type != "azure" else None
     }
+    if not isinstance(provider, AzureProvider):
```

Comment:
before it had a lot of IF-conditions, and not it got one more inside Azure. so now it is more readable